tests/Client/UpdaterTest.php

```diff
@@ -1084,15 +1084,18 @@ public function providerRefreshException(): array
     public function testFileNotFoundExceptions(string $fixtureName, string $fileName, array $expectedUpdatedVersions): void
     {
         $updater = $this->getSystemInTest($fixtureName);
+        // Depending on which file is removed from the server, the update
+        // process will error out at various points. That's fine, because we're
+        // not trying to complete the refresh.
         $this->serverStorage->removeRepoFile($fileName);
         try {
             $updater->refresh();
+            $this->fail('No RepoFileNotFound exception thrown');
         } catch (RepoFileNotFound $exception) {
-            $this->assertSame("File $fileName not found.", $exception->getMessage());
-            $this->assertClientFileVersions($expectedUpdatedVersions);
-            return;
+            // We don't have to do anything with this exception; we just wanted
```

Comment:
Maybe mention the exception message comes from `TestRepo` so there is not point in asserting a message produced by the test class.

Reply:
Done.